website/docs/d/scc_posture_scan_summaries.html.markdown

```diff
@@ -0,0 +1,108 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scan_summaries"
+description: |-
+  Get information about scan_summaries
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scan_summaries
+
+Provides a read-only data source for scan_summaries. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
1. Provides a read-only data source for scan_summaries.
as
Review information of Security and Compliance Center scan summaries. 

2. Add reference link of scan summaries